packages/blob/src/test/index.ts

```diff
@@ -13,6 +13,8 @@ describe( 'isBlobURL', () => {
 	} );
 
 	it( 'returns false if the url is not defined', () => {
+		// calling isBlobURL without a URL is not type compliant, so ignore it
+		// @ts-ignore
```

Comment:
Can we use `@ts-expect-error` everywhere that we _know_ the types are wrong (and expect it)?

```suggestion
		// @ts-expect-error: This is not a valid call according to types.
```

Ideally we can target only the violating line:

```ts
expect(
// @ts-expect-error
isBlobURL()
).toBe( … )
```